mlir-tensorrt/executor/lib/Runtime/Backend/Lua/LuaRuntime.cpp

```diff
@@ -502,12 +502,138 @@ static Status validateArgsTypesAgainstFuncArgs(const RuntimeValue *runArg,
   return getOkStatus();
 }
 
+[[maybe_unused]] static constexpr int MEMREF_FIXED_FIELDS =
```

Comment:
For constant integers, our convention is to use `int64_t` and naming as follows:

```
static constexpr int64_t kMemRefFixedFields = ...
```

Reply:
This is not present in the final code anymore.